### DISCUSSION TITLE
Diablo immortal is able to log in under SIP turned off, once SIP turned on, cannot log in nor start the game

### DISCUSSION THREAD

**Discussion by huoyingzuibang** [PlayCover/PlayCover#819] [Q&A]
Diablo immortal is able to log in under SIP turned off, once SIP turned on, cannot log in nor start the game

tried, try sudo nvram boot-args="amfi_get_out_of_my_way=1"  no working.

tried, You can also try deleting any items related to your game in Keychain Access (type keychain access in Spotlight Search to open it) -- no working

Asking for help

**Comment by ohaiibuzzle** [comment]
You can try with Nightly 250 & PlayChain to see if it works. Keep in mind that game support is on a game-by-game basis and don't expect every one of them to flawlessly work

**Comment by nesk** [comment]
What's the state of PlayChain? [There was a big announcement a year before,](https://www.reddit.com/r/macgaming/comments/10fc6o2/big_news_from_playcover_team/) but it doesn't seem to be stable up to this date? Should we keep downloading nightlies to avoid disabling SIP?
